src/components/implementation/no_interface/cFE_booter/Makefile

```diff
@@ -0,0 +1,12 @@
+# C_OBJS=cFE_entrypoint.o cFE_stub.o osapi.o osfiles.o osloader.o osqueue.o ostask.o ostimer.o osnetwork.o psp.o scheddev/sl.c scheddev/sl_mod_fprr.c
+ASM_OBJS=
+COMPONENT=cFE_booter.o
+INTERFACES=cFE
+DEPENDENCIES=capmgr
+IF_LIB:=./composite_cFE.o ./cFE_fs.o $(wildcard test/*.o)
+ADDITIONAL_LIBS=-lcobj_format -lcos_kernel_api -lcos_defkernel_api -lsl_capmgr -lsl_sched -lheap -lsl_lock
```

Comment:
I've added a macro for `-lcos_kernel_api -lcos_defkernel_api -lsl_capmgr -lsl_sched` = LIBSLCAPMGR, you can probably use that.

Reply:
Awesome! That's very convenient